backend/tournesol/admin.py

```diff
@@ -197,17 +217,27 @@ class PollAdmin(admin.ModelAdmin):
         'algorithm',
         'entity_type',
         'get_n_criterias',
+        'get_n_comparisons',
+        'get_n_comparisons_per_criteria',
     )
     list_filter = (
         'algorithm',
         'entity_type'
     )
     inlines = (CriteriasInline,)
 
-    @admin.display(description="Nb of criterias")
+    @admin.display(description="# criterias")
     def get_n_criterias(self, obj):
         return obj.criterias.count()
 
+    @admin.display(description="# comparisons")
+    def get_n_comparisons(self, obj):
+        return obj.comparisons.count()
```

Comment:
These new additions are OK for now, but we should keep in mind we are generating extra 3 extra SQL queries for each poll.

This is clearly manageable for now because we have only 2 polls in production. I'll try to fix this in a next PR.